en/api.md

```diff
@@ -6438,28 +6505,28 @@ Ready.
 
 Manage a physical zone.
 
-> Any node is in a physical zone, or "zone" for short. The meaning of a
-> physical zone is up to you: it could identify e.g. a server rack, a network,
-> or a data centre. Users can then allocate nodes from specific physical
-> zones, to suit their redundancy or performance requirements.
->
-> This functionality is only available to administrators. Other users can view
-> physical zones, but not modify them.
+Any node is in a physical zone, or "zone" for short. The meaning of a
```

Comment:
I'm not sure if this is in our purview, but maybe: *Every node must belong to a physical zone.*

Reply:
yeah, thanks. I'll fix this, but you're right, it's all generated automagically from the MAAS code and it needs to be fixed at source. There are lots of things I'd like to change - I should look into updating it there.